main.c

```diff
@@ -371,9 +371,50 @@ int check_winning_condition(
     int last_coordinate[2],
     int lives
 ) {
-    // Hint: return the updated lives count
-    return lives;
-}
+    //If the Frogger lands on water, decrease lives.
+    if (board[last_coordinate[X]][last_coordinate[Y]].type == WATER)
```

Comment:
more of a suggestion, but you can assign this to a variable to reduce the length of the code:

```suggestion
    struct board_tile tile = board[last_coordinate[X]][last_coordinate[Y]];
    
    //If the Frogger lands on water, decrease lives.
    if (tile.type == WATER)
```

subsequently just refer to the `tile`